core/src/main/java/org/mule/runtime/core/routing/IdempotentMessageFilter.java

```diff
@@ -38,18 +41,15 @@
  * <b>EIP Reference:</b> <a href="http://www.eaipatterns.com/IdempotentReceiver.html">
  * http://www.eaipatterns.com/IdempotentReceiver.html</a>
  */
-public class IdempotentMessageFilter extends AbstractFilteringMessageProcessor implements Initialisable, Disposable {
+public class IdempotentMessageFilter extends AbstractInterceptingMessageProcessor implements Initialisable, Disposable {
```

Comment:
Is it possible to use what MG used for making DB not intercepting anymore?. This will bring problems with the interception API and debugger.